.github/workflows/check.yml

```diff
@@ -0,0 +1,103 @@
+name: Check Terraform Configuration
+
+on:
```

Comment:
```suggestion
on:
  push:
```

Reply:
If you're going to do push then pull_request is probably pointless as it'll have been run for that code anyway?

Also, is "Require branches to be up to date before merging" because if not there's still a chance the merge will break things... Might want to consider merge queues for these checks.

Reply:
You're right, yes.

Reply:
Merge queues are out of scope for this particular PR, but we probably should.